extensions/youtube/package.json

```diff
@@ -80,19 +80,19 @@
       "type": "dropdown",
       "title": "Grid Item Size",
       "required": false,
-      "default": "medium",
+      "default": "5",
```

Comment:
The downside here is that everyody who has already a non default have a wrong value, any maybe crash. Can you check that upfront

Reply:
Not sure what you mean here. The extension will not crash but I believe that the preference will be reset to the default value (which is 5 now) for everyone. They will have to reselect their preference.

Reply:
Yeah. I check the the code again. Ignore this comment.